include/librealsense2/h/rs_context.h

```diff
@@ -27,8 +27,9 @@ rs2_context* rs2_create_context(int api_version, rs2_error** error);
 * \param[in] api_version Users are expected to pass their version of \c RS2_API_VERSION to make sure they are running the correct librealsense version.
 * \param[in] json_settings Pointer to a string containing a JSON configuration to use, or null if none
 *            Possible settings:
-*                dds-discovery - (bool) false to disable DDS discovery; defaults to true (requires BUILD_WITH_DDS)
-*                dds-domain - (int) the number of the domain discovery is on (requires BUILD_WITH_DDS)
+*                dds               - (requires BUILD_WITH_DDS) false (bool) to disable DDS; otherwise the DDS settings:
```

Comment:
I think `(bool, requires BUILD_WITH_DDS)` is more clear. Add "default true"

Reply:
It defaults to `{}`. But point taken, will change.